mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -176,23 +175,16 @@ SYCLOpAsmInterface::getAlias(mlir::Type Type, llvm::raw_ostream &OS) const {
            << "_";
         return AliasResult::OverridableAlias;
       })
-      .Case<mlir::sycl::GetScalarOpType, mlir::sycl::MinimumType,
-            mlir::sycl::MaximumType, mlir::sycl::TupleValueHolderType>(
-          [&](auto Ty) {
-            OS << "sycl_" << decltype(Ty)::getMnemonic() << "_"
-               << Ty.getDataType() << "_";
-            return AliasResult::FinalAlias;
-          })
       .Case<mlir::sycl::SwizzledVecType>([&](auto Ty) {
         const auto VecTy = Ty.getVecType();
         OS << "sycl_" << decltype(Ty)::getMnemonic() << "_"
            << VecTy.getDataType() << "_" << VecTy.getNumElements() << "_";
         return AliasResult::OverridableAlias;
       })
-      .Case<mlir::sycl::TupleCopyAssignableValueHolderType>([&](auto Ty) {
+      .Case<mlir::sycl::MinimumType, mlir::sycl::MaximumType>([&](auto Ty) {
```

Comment:
[nit] Place it before the case of `SwizzledVecType`.
```suggestion
      .Case<mlir::sycl::MaximumType, mlir::sycl::MinimumType>([&](auto Ty) {
```

Reply:
I placed it before the `MultiPtr` case

Reply:
even better, thanks.